pkg/util/json/json_test.go

```diff
@@ -603,6 +603,59 @@ func TestJSONExists(t *testing.T) {
 	}
 }
 
+func TestJSONStripNulls(t *testing.T) {
+	testcases := []struct {
+		input    string
+		expected string
+	}{
+		{`null`, `null`},
+		{`1`, `1`},
+		{`"a"`, `"a"`},
+		{`{}`, `{}`},
+		{`[]`, `[]`},
+		{`[1, "a", null, {"a":null}]`, `[1, "a", null, {}]`},
+		{`{"a":[null], "b":null, "c":{"a":null}}`, `{"a":[null], "c":{}}`},
+	}
+	for _, tc := range testcases {
+		expectedResult, err := ParseJSON(tc.expected)
+		if err != nil {
+			t.Fatal(err)
+		}
+		runFuncCtr := func(encoded bool) func(*testing.T) {
```

Comment:
We do have `runDecodedAndEncoded` for this - is there are reason that didn't do what you needed?